pkgs/applications/networking/browsers/qutebrowser/default.nix

```diff
@@ -31,14 +31,26 @@ let
 
 in mkDerivationWith python3Packages.buildPythonApplication rec {
   pname = "qutebrowser";
-  version = "1.14.1";
+  version = "2.0.0-pre";
 
   # the release tarballs are different from the git checkout!
-  src = fetchurl {
-    url = "https://github.com/qutebrowser/qutebrowser/releases/download/v${version}/${pname}-${version}.tar.gz";
-    sha256 = "15l7jphy1qjsh6y6kd5mgkxsl6ymm9564g1yypa946jbyrgi8k2m";
+  #src = fetchurl {
+  #  url = "https://github.com/qutebrowser/qutebrowser/releases/download/v${version}/${pname}-${version}.tar.gz";
+  #  sha256 = "15l7jphy1qjsh6y6kd5mgkxsl6ymm9564g1yypa946jbyrgi8k2m";
+  #};
```

Comment:
```suggestion
```

Reply:
This is a draft PR. This is commented out because I need to temporarily use the sources from git and I will change it back to fetchurl once release tarballs exist (which means before this is merged).